docs/tests/sitemap.test.ts

```diff
@@ -0,0 +1,11 @@
+import * as fs from 'fs';
+
+test('Sitemap Snapshot', () => {
+  const sitemapData = fs.readFileSync('./public/sitemap.xml', 'utf8');
+  const regexOfLinks = new RegExp('(?<=>).+?(?=</loc)', 'gmi'); // This Regex is to match all the content between `>` and end with `</loc`.
```

Comment:
This is just me being really nitty, but personally prefer var names to not have prepositions; what do you think about

```suggestion
  const linkRegex = new RegExp('(?<=>).+?(?=</loc)', 'gmi'); // This Regex is to match all the content between `>` and end with `</loc`.
```

or just back to `regex` now that you added a comment?